package-lock.json

```diff
@@ -3640,35 +4284,37 @@
             "find-up": "^4.0.0"
           }
         },
-        "prop-types": {
-          "version": "15.7.2",
-          "bundled": true,
-          "dev": true,
-          "requires": {
-            "loose-envify": "^1.4.0",
-            "object-assign": "^4.1.1",
-            "react-is": "^16.8.1"
-          }
-        },
         "punycode": {
           "version": "2.1.1",
           "bundled": true,
           "dev": true
         },
-        "react-is": {
-          "version": "16.13.1",
+        "react": {
```

Comment:
seems odd to be including **`react`** as a dependency/devDependency here ... what is the usecase? 🤷‍♂️

Reply:
Appears to be a `devDependency` of `tsd`: https://github.com/SamVerschueren/tsd/blob/5347e25e2ffb28739ea3e877faddc22d2e9e49b9/package.json#L62
Totally get that they might want to test their code against `react`. 
But then why does it have to appear in our dependency tree? 🙄 
When we `npm install` do we _need_ to have the grandchild `devDependencies` for our tests (`tsd`) to pass? 💭

Reply:
Good catch! I wouldn't expect their devDependencies to show up in ours. I'm currently on holiday without my laptop (bold, I know) so I can have a look at this in 2 weeks

Reply:
Good catch, I wouldn't expect their devDependencies to show up in this tree. I'm currently on holiday without my laptop so I can't dig in deeper right now. After the 12th of December I'll have a look as to why this dependency is showing up here